app/_hub/moesif/kong-plugin-moesif/_metadata.yml

```
@@ -23,6 +23,9 @@ privacy_policy_url: https://www.moesif.com/privacy?utm_medium=docs&utm_campaign=
 
 terms_of_service_url: https://www.moesif.com/terms?utm_medium=docs&utm_campaign=partners&utm_source=kong
 
+free: true
+plus: true
```

`plus: true` is a Konnect-specific tier for pay-as-you go customers. Since your plugin is not bundled with Konnect, this doesn't apply.
```suggestion
plus: false
```

---

Fixed in latest commit